doc/modules/svm.rst

```diff
@@ -705,7 +705,7 @@ computations. These libraries are wrapped using C and Cython.
     - `LIBSVM: a library for Support Vector Machines
       <http://www.csie.ntu.edu.tw/~cjlin/papers/libsvm.pdf>`_
 
-    - `LIBLINEAR -- A Library for Large Linear Classification
```

Comment:
The name of the paper / project is "LIBLINEAR -- A Library for Large Linear Classification", and should probably be cited as such.